<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyManagerForm</class>
 <widget class="QWidget" name="KeyManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeyManager</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFile">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>65</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNewFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFilePath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveFile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保 存</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbkmFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFolder">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>5</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件夹路径:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFolderPath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件夹</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>使用帮助</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="twKey">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>231</width>
     <height>691</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>项目名称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>创建时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>私钥数量:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>备注:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>创建时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>备注:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>名称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>序号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>820</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>助记词:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>680</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>公钥:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>私钥:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>750</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>地址:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leProjectName">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leProjectCreationTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKeyCount">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKeyName">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKeyNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKeyCreationTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteProjectNote">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteKeyNote">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>560</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ptePrivateKey">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>630</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ptePublicKey">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>700</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteKeyAddress">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>770</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteKeyMnemonic">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>840</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>840</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单击列表项复制地址,双击复制私钥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKeyAdd">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新增密钥</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbKeyUsed">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已启用</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbKeyDisabled">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>废弃</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFilter">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>项目信息</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>251</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>密钥信息</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>btnOpenFile</zorder>
  <zorder>btnNewFile</zorder>
  <zorder>labelFilePath</zorder>
  <zorder>btnSaveFile</zorder>
  <zorder>cbkmFile</zorder>
  <zorder>btnOpenFolder</zorder>
  <zorder>label_2</zorder>
  <zorder>labelFolderPath</zorder>
  <zorder>btnHelp</zorder>
  <zorder>twKey</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>leProjectName</zorder>
  <zorder>leProjectCreationTime</zorder>
  <zorder>leKeyCount</zorder>
  <zorder>leKeyName</zorder>
  <zorder>leKeyNumber</zorder>
  <zorder>leKeyCreationTime</zorder>
  <zorder>pteProjectNote</zorder>
  <zorder>pteKeyNote</zorder>
  <zorder>ptePrivateKey</zorder>
  <zorder>ptePublicKey</zorder>
  <zorder>pteKeyAddress</zorder>
  <zorder>pteKeyMnemonic</zorder>
  <zorder>label_15</zorder>
  <zorder>btnKeyAdd</zorder>
  <zorder>cbKeyUsed</zorder>
  <zorder>cbKeyDisabled</zorder>
  <zorder>cbFilter</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
